cmd/instance/instance_list.go

```diff
@@ -59,39 +60,41 @@ If you wish to use a custom format, the available fields are:
 		}
 
 		ow := utility.NewOutputWriter()
-		for _, instance := range instances {
-			ow.StartLine()
+		if len(instances) == 0 {
+			fmt.Println("region", client.Region, ".No instances found in this region")
+		} else {
```

Comment:
We don't need the else block here. If it's not 0, rest of the code is executed either way

Reply:
Removed the else block 👍🏼